components/modal/index.css

```diff
@@ -130,17 +131,15 @@
 /** @unofficial */
 .spectrum-Modal--fullscreen {
 	position: fixed;
-	inset-inline-start: var(--spectrum-modal-fullscreen-margin);
-	inset-block-start: var(--spectrum-modal-fullscreen-margin);
-	inset-inline-end: var(--spectrum-modal-fullscreen-margin);
-	inset-block-end: var(--spectrum-modal-fullscreen-margin);
+	inset-block: var(--mod-modal-fullscreen-margin, var(--spectrum-modal-fullscreen-margin));
+	inset-inline: var(--mod-modal-fullscreen-margin, var(--spectrum-modal-fullscreen-margin));
```

Comment:
Are there any concerns with consolidating these? This isn't a necessary change, so I'm totally open to reverting.